lib/venue.rb

```diff
@@ -1,2 +1,38 @@
 class Venue
+
+  attr_reader :name, :capacity, :patrons
+
+  def initialize (name_arg, capacity_arg)
+    @name = name_arg
+    @capacity = capacity_arg
+    @patrons = []
+  end
+
+  def add_patron (name)
+    patrons.push(name)
+  end
+
+  def yell_at_patrons
+    capitilized_names = []
+    patrons.each do |patron_name|
+      capitilized_names.push(patron_name.upcase)
+    end
+    return capitilized_names
+  end
+
+  def over_capacity?
+    if patrons.count > capacity
+      return true
```

Comment:
Nice use of the `if/else` here but you can just write 
```
patrons.count > capacity
```
because you are using the `>` operator here it will automatically return a boolean